code/modules/awaymissions/mission_code/ruins/deepstorage.dm

```diff
@@ -282,6 +282,27 @@
 	name = "portal"
 	desc = "Good luck."
 
+// loot spawners
+
+/obj/effect/spawner/lootdrop/deepstorage_reward
+	name = "warehouse fashion reward"
+	lootcount = 1
+	loot = list(
+			/obj/item/storage/box/syndie_kit/chameleon,
+			/obj/item/clothing/suit/pimpcoat,
+			/obj/item/melee/skateboard/hoverboard,
+			/obj/item/clothing/glasses/sunglasses/yeah
+	)
+
+/obj/effect/spawner/lootdrop/deepstorage_reward/main
+	name = "warehouse main reward"
+	loot = list(
+			/obj/item/storage/belt/champion/wrestling,
+			/obj/item/storage/box/weaver_kit,
+			/obj/item/gun/medbeam,
+			/obj/item/storage/lockbox/experimental_weapon
+	)
+
```

Comment:
Could you swap these out with the new random spawner type? Should be as simple as:


```suggestion
/obj/effect/spawner/random/deepstorage_reward
	name = "warehouse fashion reward"
	loot = list(
			/obj/item/storage/box/syndie_kit/chameleon,
			/obj/item/clothing/suit/pimpcoat,
			/obj/item/melee/skateboard/hoverboard,
			/obj/item/clothing/glasses/sunglasses/yeah
	)

/obj/effect/spawner/random/deepstorage_reward/main
	name = "warehouse main reward"
	loot = list(
			/obj/item/storage/belt/champion/wrestling,
			/obj/item/storage/box/weaver_kit,
			/obj/item/gun/medbeam,
			/obj/item/storage/lockbox/experimental_weapon
	)

```

And then changing the types in the map.